lib/presentation/view/home.dart

```diff
@@ -1,15 +1,237 @@
+                    ),
+                  ),
+                ],
+                title: Padding(
+                  padding: EdgeInsets.symmetric(
+                    horizontal: MediaQuery.of(context).size.width *
+                        Dimensions.fivePercent,
+                  ),
+                  child: const CompanyLogo(),
+                ),
+                expandedHeight: Dimensions.sliverAppbarExtendedHeight,
+                flexibleSpace: FlexibleSpaceBar(
+                  collapseMode: CollapseMode.parallax,
+                  centerTitle: true,
+                  title: constraints.crossAxisExtent >= Dimensions.smallScreen
+                      ? const Text(TextConstants.slogan)
+                      : null,
+                  background: Stack(
+                    fit: StackFit.expand,
+                    children: [
+                      Image.asset(
+                        AssetConstants.sliverBackground,
+                        fit: BoxFit.cover,
+                      ),
+                      const ColoredBox(
+                        color: Palette.darkOverlay,
+                      ),
+                    ],
+                  ),
+                ),
+              );
+            },
+          ),
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: EdgeInsets.symmetric(
+                horizontal: (MediaQuery.of(context).size.width *
+                    Dimensions.fivePercent),
+              ),
+              child: Container(
+                padding: Dimensions.topPadding,
+                decoration: const BoxDecoration(
+                  border: BorderDirectional(
+                    bottom: BorderSide(
+                      color: Colors.black54,
+                    ),
+                  ),
+                ),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    const Text(
+                      TextConstants.randomRecipesTitle,
+                      style: TextStyle(
+                        fontSize: Dimensions.bigFontSize,
+                        fontWeight: FontWeight.w400,
+                        color: Colors.black54,
+                      ),
+                    ),
+                    GestureDetector(
+                      key: Keys.refreshRecipes,
+                      onTap: () => recipesController
+                          .getRandomRecipes(ApiConstants.numberOfRandomRecipes),
+                      child: const Icon(
+                        Icons.refresh,
+                        color: Colors.black54,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+          recipesController.obx(
+            (state) {
+              return SliverPadding(
+                padding: EdgeInsets.symmetric(
+                  horizontal: MediaQuery.of(context).size.width *
```

Comment:
You could replace MediaQuery.of(context).size.width to Get.width

Reply:
Done